ui/src/app/shared/pipes/cesure.pipe.ts

```diff
@@ -0,0 +1,32 @@
+import {Pipe, PipeTransform} from '@angular/core';
+
+@Pipe({
+  name: 'cesureLeft'
+})
+export class CesureLeftPipe implements PipeTransform {
```

Comment:
Maybe rename this Pipe to TruncateLeftPipe to make the meaning a bit clearer?

Reply:
Merged with the first pipe (direction is a parameter)

Reply:
+ 'trail' (…) is a parameter